### PR TITLE
Add ownCloud version dependency and make it so JS/CSS only load on the files app

### PR DESCRIPTION
**Pull request by mdusher**
I've been using this app with ownCloud with some slight modifications. 

I have added the ownCloud version dependency to appinfo/info.xml and made it so the CSS/JS only loads when you are on the Files app (this was to avoid collisions with CSS libraries used in other apps). I also made the installation instructions a little clearer where I tripped up with them during my installation!

Everything here is compatible with NextCloud as well. I've tested this in ownCloud 10.0.3.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,15 +23,17 @@ For more information, see the blog post [Nextcloud in Digital Imaging](https://n
 
 ### Installation
 
-On your Nextcloud, simply navigate to Apps > Multimedia > DICOM Viewer, and enable it.
+Build the app using the instructions below and then on Nextcloud, simply navigate to Apps > Multimedia > DICOM Viewer, and enable it.
```
Comment (ayselafsar) [inline README.md]: No need to build for end users to use the released version. It is available under Apps > Multimedia on Nextcloud.

File: appinfo/info.xml [hunk 2]
```diff
@@ -16,6 +16,7 @@ See [README](https://github.com/ayselafsar/dicomviewer) for a list of full featu
     <bugs>https://github.com/ayselafsar/dicomviewer/issues</bugs>
     <dependencies>
         <nextcloud min-version="12" max-version="14"/>
+        <owncloud min-version="10" max-version="10"/>
```
Comment (ayselafsar) [inline appinfo/info.xml]: We officially do not support for owncloud.

**Comment by ayselafsar** [comment]
Could you please update this PR based on my comments?

Thanks for your contribution!

**Comment by Lars1963** [comment]
Could being closed as of PR's #35 and #36, but without the oc-comments (as not being supported).

**Pull request closed without merging**